workshop_materials.md

```diff
@@ -0,0 +1,104 @@
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/intro_cellprofiler_workshop/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/intro_cellprofiler_workshop)
+
+### Introduction to Fiji
+
+[Fiji](https://fiji.sc) is a distribution of the [ImageJ](https://imagej.net/software/imagej/) open-source software package for analyzing scientific images. This workshop covers the basic concepts of image analysis (e.g. pixels, bit-depth) and then key Fiji concepts including the GUI layout, scale bars, measurements, and stacks. Finally, selected analysis applications/workflows are demonstrated, including segmentation, colocalization, and tracking. 
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/fiji_workshops/IntroFiji.html)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/fiji_workshops)
+
+### Intermediate Fiji: Macros & Scripting
+`<materials in progress>`
+
+### Introduction to napari
+
+[napari](https://napari.org) is a Python image visualization and annotation application. This workshop is an introduction to napari and the napari ecosystem. It covers key concepts such as the GUI layout, viewer and layer models, as well as more advanced concepts such as the ability to bidirectionally interact with napari from Python code and make simple GUI widgets.  
+Note: It is intended for people who already have some familiarity with Python.
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/intro-napari-workshop/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/intro-napari-workshop)
+
+### Introduction to QuPath
+
+[QuPath](https://qupath.github.io) is a Java-based image visualization, annotation, and analysis package. This workshop begins with an introduction to the software and cover key concepts such as projects, images, classes, annotations, detections, followed by an example of segmentation and classification.
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/intro_qupath_workshop/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/intro_qupath_workshop)
+
+## Python-related workshops
+
+Note: These workshops are intended for people who already have some familiarity with Python.
+
+### (Bio)Image Processing with Python
+
+This workshop will cover reading image files into Python to visualize and manipulate the image data using scikit-image, the most prominent image processing library for Python. Then it covers building a basic bioimage processing pipeline to apply basic filters, threshold, and segment objects in an image.  
+
+[Workshop materials](https://thejacksonlaboratory.github.io/image-processing-ia/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/image-processing-ia)
+
+### OMERO for Python Users
```

Comment:
Not sure what order to put this one in, before/after the zarr/dask?

Reply:
Where it is makes sense to me